### PR TITLE
Hiv 610 plates plugin

### PR DESCRIPTION
**Pull request by amazimbe**

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
-        onsubmit(e) {
-          // Insert content when the window form is submitted
-          const kebabbyString = _.kebabCase(e.data.title);
-          editor.insertContent(kebabbyString);
-        }
-      });
-    }
+    tooltip: 'Plate',
+    icon: 'plates',
+    menu: menu,
```
Comment (stephanebruckert) [inline src/plugin.js]: `menu: menu,` -> `menu,`

File: src/context.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    i += 1;
+  }
+
+  i = 1;
+  while (i <= y) {
+    yData.push(String.fromCharCode(64 + i));
+    i += 1;
+  }
+
+  return { xData: xData, yData: yData };
```
Comment (stephanebruckert) [inline src/context.js]: `return { xData, yData };`

**Review by stephanebruckert — CHANGES_REQUESTED** [review]
Some packages have updates:
```
➜  tinymce-plates git:(HIV-610-Plates-Plugin) npm outdated
Package                 Current  Wanted  Latest  Location
babel-loader              6.4.1   6.4.1   7.1.2  plates
eslint                   3.19.0  3.19.0   4.6.1  plates
eslint-config-standard    6.2.1   6.2.1  10.2.1  plates
eslint-plugin-standard    2.3.1   2.3.1   3.0.1  plates
webpack                   2.7.0   2.7.0   3.5.6  plates
webpack-dev-server       1.16.5  1.16.5   2.7.1  plates
```

**Comment by amazimbe** [comment]
this has all been generated and except  in the most obvious cases, I'd
rather not touch most of this.

**Pull request merged**